docs/articles/documentation/reference/test-api/testcontroller/expect/README.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: docs
+title: t.expect Method
+permalink: /documentation/reference/test-api/testcontroller/expect/
+---
+# t.expect Method
+
+This method accepts the actual value.
+
+You can pass a value, a [Selector's DOM node state property](../../../../guides/basic-guides/select-page-elements.md#define-assertion-actual-value)
+or a [client function](../../../../guides/basic-guides/obtain-data-from-the-client.md) promise.
+TestCafe automatically waits for node state properties to obtain a value and for client functions to execute. See [Smart Assertion Query Mechanism](../../../../guides/basic-guides/assert.md#smart-assertion-query-mechanism) for details.
+
+The value passed as the `actual` parameter needs to be of the data type supported by the assertion method.
```

Comment:
```suggestion
The value passed as the `actual` parameter needs to an acceptable data type.
```

Reply:
@dirk-pieterse thank you for the suggestion! But we do need to highlight that the data type must be supported by assertion methods. Simply saying "acceptable data type" is not a good option here

Reply:
When you write it this way, there can be multiple interpretations of "supported by the assertion method".

Reply:
```suggestion
The value passed as the `actual` parameter should be a data type the assertion method accepts.
```